src/components/TablePerson/TablePerson.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+export const TablePerson: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames('', {
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames('', {
+            'has-text-danger': sex === 'f',
+          })}
+        >
+          {name}
+        </Link>
```

Comment:
create the <PersonLink person={person} /> and use it for all existing people;
<img width="652" alt="image" src="https://github.com/user-attachments/assets/e4f6ffe5-4de5-4833-999e-443bb6b49002">

And the same for mother and father